com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -515,17 +515,19 @@ internal static void NetworkBehaviourUpdate(NetworkManager networkManager)
                     s_Touched.Clear();
                     for (int i = 0; i < networkManager.ConnectedClientsList.Count; i++)
                     {
+                        s_TouchedThisClient.Clear();
```

Comment:
This is the only integration point (so far) ...network variables are filtered based on the Interest nodes

Reply:
Ancient code I forgot to update - fixing

Reply:
I think we should explore how this will fit into the bigger picture. With the snapshot system we might change how NetworkVariables are transferred so we should make sure that the interest system fits into that architecture.

I wonder whether RPC support is something we would want? Probably yes?

Reply:
Yeah, I started looking at how to hook into the RPC filter.  I'm assuming (duh) I want to do this ahead of where it gets queued.  Since the RPC is always directed to an object, it should be quite doable to ahead of the Server->Client RPC to drop the message if the target object is not Interest-covered for the target network id.  I could use a bit of guidance @MFatihMAR @NoelStephensUnity next week

Reply:
`NetworkBehaviour`'s `__beginSendServerRpc()`/`__endSendServerRpc()` and `__beginSendClientRpc()`/`__endSendClientRpc()` methods are places to integrate this — but yeah, we could chat next week for sure.